main.js

```diff
@@ -0,0 +1,46 @@
+let inputBox = document.getElementById("inputValue");
+let noTasks = document.getElementById('noTasks');
+
+// Getting today's date
+  n =  new Date();
+  y = n.getFullYear();
+  m = n.getMonth() + 1;
+  d = n.getDate();
+  document.getElementById("currentDate").innerHTML = m + "/" + d + "/" + y; 
+  window.onload = inputBox.focus();  
+  function addTask() { 
+  let inputValue = document.getElementById("inputValue").value; 
+  let deadlineDate = document.getElementById("deadlineDate").value; 
```

Comment:
leave empty spaces between different definition, for example if you were defining variables and suddenly decided to define a function, leave an empty line between them and so on. Also indentation, that means a code inside a function should have 4 spaces or 2 from the left so it indent inside the function or [code block](https://www.google.com/search?q=what+is+code+block+in+js&sxsrf=AOaemvKTHiQgWZkI_ui7xIsIYPYWx-YKEg%3A1636016478493&ei=XqGDYfSsHbiI9u8PgLKdsA8&oq=what+is+code+block+in+js&gs_lcp=Cgdnd3Mtd2l6EAMyBQgAEIAEMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQyBQgAEIAEMgUIABCABDIFCAAQgAQyBggAEBYQHjoHCAAQRxCwAzoHCAAQsAMQQ0oECEEYAFCyLFjrMGD1MWgBcAJ4AIABxgOIAcYDkgEDNC0xmAEAoAEByAEKwAEB&sclient=gws-wiz&ved=0ahUKEwj006-orP7zAhU4hP0HHQBZB_YQ4dUDCA4&uact=5)

you can read more here about [code formatting](https://torquemag.io/2019/07/code-formatting-guide/). and I highly recommend you use [prettier](https://prettier.io/) extension in VSCode, it saves you a lot of time
```suggestion
  document.getElementById("currentDate").innerHTML = m + "/" + d + "/" + y; 
  window.onload = inputBox.focus();  
  
  function addTask() { 
      let inputValue = document.getElementById("inputValue").value; 
      let deadlineDate = document.getElementById("deadlineDate").value; 
```